bindings/go-apiv2/go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/hyperledger/fabric-protos-go-apiv2
 go 1.17
 
 require (
-	google.golang.org/grpc v1.46.2
-	google.golang.org/protobuf v1.28.0
+	google.golang.org/grpc v1.57.0
```

Comment:
Fabric core repository is having an issue with the grpc update, I meant to look into that before merging this one, let me go back and look into that further...

Reply:
This should not impact Fabric core. This is an update to fabric-protos-apiv2, which is not used by Fabric core. fabric-gateway does use this package, but fabric-gateway is also using the same gRPC version and tests are passing fine:

https://github.com/hyperledger/fabric-gateway/blob/320d90ca584b59cf5628b01420403ffbc5cd7b61/go.mod#L14

Reply:
Ok cool, will merge.
I did find the issue with grpc update in core fabric so I will resolve that separately.